src/pywifes.py

```diff
@@ -732,7 +732,7 @@ def subtract_overscan(inimg, outimg,
     detsize_str = '[%d:%d,%d:%d]' % (1, ny, 1, nx)
     outfits[data_hdu].header.set('DETSIZE', detsize_str)
     outfits[data_hdu].header.set('CCDSIZE', detsize_str)
-    #outfits[data_hdu].header.set('CCDSEC',  detsize_str)
+    #outfits[data_hdu].header.set('DETSEC',  detsize_str)
```

Comment:
I know ... And there are too many commented lines everywhere but I left this one (and some others) exactly because of what you said about the configuration options, just in case this helps us in a later stage when performing a deeper cleaning of the scripts.